packages/xrpc-server/src/rate-limiter.ts

```diff
@@ -109,6 +110,30 @@ export class RateLimiter implements RateLimiterI {
       }
     }
   }
+
+  delete = async (
+    ctx: XRPCReqContext,
+    opts?: { calcKey?: CalcKeyFn },
+  ): Promise<void> => {
+    const key = opts?.calcKey ? opts.calcKey(ctx) : this.calcKey(ctx)
+    if (key === null) {
+      return
+    }
+
+    try {
+      await this.limiter.delete(key)
+    } catch (err) {
+      logger.error(
```

Comment:
Since this doesn't get handled as middleware, i wonder if it's better to let the error bubble up to the caller? Otherwise the caller has no way of knowing if the limit was actually reset